api/config/environment.go

```diff
@@ -36,8 +38,9 @@ type EnvironmentConfig struct {
 	DeploymentTimeout time.Duration `yaml:"deployment_timeout"`
 	NamespaceTimeout  time.Duration `yaml:"namespace_timeout"`
 
-	MaxCPU    string `yaml:"max_cpu"`
-	MaxMemory string `yaml:"max_memory"`
+	MaxCPU                    string                    `yaml:"max_cpu"`
+	MaxMemory                 string                    `yaml:"max_memory"`
+	TopologySpreadConstraints TopologySpreadConstraints `yaml:"topology_spread_constraints"`
```

Comment:
Just realised that in Merlin, this is going to be a part of each environment's individual config. Whereas in Turing, it is a common deployment config: https://github.com/caraml-dev/turing/blob/main/api/turing/config/config.go#L233

Something we can think about unifying, when we move the environments to MLP.

Reply:
Yep, the structure of the configs in both apps can be unified and made more consistent 😅